src/js/claims-status/components/Breadcrumbs.jsx

```diff
@@ -1,14 +1,59 @@
 import React from 'react';
+import { buildMobileBreadcrumb, debouncedToggleLinks } from '../../utils/breadcrumb-helper';
 
 class Breadcrumbs extends React.Component {
+  componentDidMount() {
+    buildMobileBreadcrumb('va-breadcrumbs-claims', 'va-breadcrumbs-claims-list');
+
+    window.addEventListener('DOMContentLoaded', () => {
+      buildMobileBreadcrumb.bind(this);
+    });
+
+    window.addEventListener('resize', () => {
+      debouncedToggleLinks('va-breadcrumbs-claims-list');
+      debouncedToggleLinks.bind(this);
+    });
+  }
+
+  componentDidUpdate() {
+    buildMobileBreadcrumb('va-breadcrumbs-claims', 'va-breadcrumbs-claims-list');
+
+    window.addEventListener('DOMContentLoaded', () => {
+      buildMobileBreadcrumb.bind(this);
+    });
+
+    window.addEventListener('resize', () => {
+      debouncedToggleLinks('va-breadcrumbs-claims-list');
+      debouncedToggleLinks.bind(this);
+    });
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('DOMContentLoaded', () => {
```

Comment:
I don't think this is removing an event listener, since you're passing in a new function that won't match any of the current event listeners. I think what we usually to is create a method on a component and then do `addEventListener('resize', this.toggleLinks)` and `removeEventListener('resize', this.toggleLinks)`

Reply:
☝️ ➕